website/src/pages/BlogPage/index.tsx

```diff
@@ -118,15 +119,27 @@ const BlogPage: React.FC = () => {
         break;
     }
   };
-
+    
   return (
     <>
-    {currentBlogDetails?.id && (
+    {currentBlogDetails?.id && Boolean(currentBlogDetails?.notHasFeatureImage) && (
+      <Metadata 
+        title={currentBlogDetails?.title} 
+        description={currentBlogDetails?.excerpt} 
+        url={currentLocation} 
+        image={`${currentOrigin}/images/blog/defaultImage.png`} 
+        isPost={true}
+        author={{ name: currentBlogDetails?.author, image: `${currentOrigin}/images/blog/authors/${currentBlogDetails?.author
+        .toLowerCase().replace(/[^\w ]+/g,'')
+        .replace(/ +/g,'-')}.png` }}
+        tags={currentBlogDetails.tags}  />
+    )}
+    {currentBlogDetails?.id && !Boolean(currentBlogDetails?.hasFeatureImage) && (
```

Comment:
This is supposed to be `notHasFeatureImage` right?

Reply:
yup, thanks for pointing out!!